modules/reference/pages/rpk/rpk-topic/rpk-topic-trim-prefix.adoc

```diff
@@ -5,14 +5,12 @@ Trim records from topics
 
 This command allows you to trim records from topics, where Redpanda
 sets the LogStartOffset for partitions to the requested offset. All segments
-whose base offset is less then the requested offset are deleted, and any records
+whose base offset is less than the requested offset are deleted, and any records
 within the segment before the requested offset can no longer be read.
 
 The `--offset/-o` flag allows you to indicate which index you want to set the
 partition's low watermark (start offset) to. It can be a single integer value
-denoting the offset, or a timestamp if you prefix the offset with an '@'. You may
-select which partition you want to trim the offset from with the `--partitions/-p`
-flag.
+denoting the offset, or it can be a timestamp if you prefix the offset with an '@'. You can select which partition you want to trim the offset from with the `--partitions/-p` flag.
```

Comment:
```suggestion
denoting the offset, or it can be a timestamp if you prefix the offset with an '@'. You can select which partition to trim the offset from using the `--partitions/-p` flag.
```